apps/game/src/sandbox/speed.rs

```diff
@@ -214,7 +214,9 @@ impl TimePanel {
             // on the right, except at the very end (for the last 'radius' pixels). And
             // when the width is too small for the radius, this messes up.
             progress_bar.push(bar_bg, Polygon::rectangle(bar_width, bar_height));
-            progress_bar.push(bar_fg, Polygon::rectangle(finished_width, bar_height));
+            if let Ok(p) = Polygon::maybe_rectangle(finished_width, bar_height) {
```

Comment:
This is the one caller of `rectangle` that crashed immediately ;)  There may be others lurking...

Reply:
It seems like it'd only crash if the width/height is less than whatever coordinate precision we're using, right?

I can imagine a `finished_width = 0` being problematic.